hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractTest.php

```diff
@@ -121,4 +121,28 @@ public function testGetActiveContractsCanReturnContractsActiveOnASpecificPeriod(
     $this->assertCount(2, $activeContracts);
   }
 
+
+  public function testGetCurrentContract() {
+    $contactParams = array("first_name" => "chrollo", "last_name" => "lucilfer");
+    $contactID =  $this->createContact($contactParams);
+
+    $params = array(
+      'position' => 'spiders boss',
+      'title' => 'spiders boss');
+    $this->createJobContract($contactID, '2015-06-01', '2015-10-01', $params);
+
+    $params = array(
+      'position' => 'spiders boss2',
+      'title' => 'spiders boss2');
+    $this->createJobContract($contactID, '2016-06-01', null, $params);
+
+    $params = array(
+      'position' => 'spiders boss3',
+      'title' => 'spiders boss3');
+    $this->createJobContract($contactID, '2014-06-01', '2014-10-01', $params);
+
+    $current_contract = CRM_Hrjobcontract_BAO_HRJobContract::getCurrentContract($contactID);
```

Comment:
Just an observation: Be careful to not mix camelCase with snake_case (I do that all the time! :( ). You used camelCase for $contacParams and $contactParams, but snake_case for $current_contract.